cmd/gaia/app/app.go

```diff
@@ -179,6 +187,9 @@ func MakeCodec() *codec.Codec {
 func (app *GaiaApp) BeginBlocker(ctx sdk.Context, req abci.RequestBeginBlock) abci.ResponseBeginBlock {
 	tags := slashing.BeginBlocker(ctx, req, app.slashingKeeper)
 
+	// mint new tokens
+	mint.BeginBlocker(ctx, app.mintKeeper)
```

Comment:
Should we call this after `distr.BeginBlocker` - if new tokens are minted "this block", they should go to whoever signed this block, whereas `distr.BeginBlocker` will allocate for last block?

Not a big deal though.

Reply:
Agreed it should be after, thanks